pkg/repometrics/vulnerabilities.go

```diff
@@ -17,11 +17,14 @@ type Scanner interface {
 func (m *Metrics) ScanVulnerabilities(ctx context.Context, s Scanner) error {
 	var err error
 
-	vulns, err := s.Scan(ctx)
+	vulnerabilities, err := s.Scan(ctx)
 	if err != nil {
 		return fmt.Errorf("scanning vulnerabilities: %w", err)
 	}
-	m.Vulnerabilities = vulns
+	// m.Vulnerabilities = vulnerabilities //we need to produce a cycloneDX file from scanning, //nolint:gofumpt
+	// and have that file being picked up in same way as externally generated setups using SBOM, Vulnerability files and VEX //nolint:gofumpt
 
+	fmt.Println("Vulnerabilities should be produced in CycloneDX format")
+	fmt.Println(vulnerabilities)
```

Comment:
Not sure if it is the responsibility of `ScanVulnerabilities` to print things?

Reply:
see comment above